engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala

```diff
@@ -618,13 +618,9 @@ object UpsertVisualizationJob {
             val externalId = expressionId
             module.getIr.preorder
               .find(_.getExternalId.contains(externalId))
-              .collect {
-                case name: Name.Literal =>
```

Comment:
why that change?

Reply:
Nowadays it is enough to query the dataflow analysis. I assume the structure of dependencies graph has changed with time